src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -549,11 +549,7 @@ nonFungibleResourceManagerFieldIdTypeSubstate.Value.NonFungibleIdType switch
                         CostingParameters = committedTransaction.Receipt.CostingParameters.ToJson(),
                         FeeDestination = committedTransaction.Receipt.FeeDestination?.ToJson(),
                         FeeSource = committedTransaction.Receipt.FeeSource?.ToJson(),
-                        EventSbors = default!, // configured later on
-                        EventSchemaEntityIds = default!, // configured later on
-                        EventSchemaHashes = default!, // configured later on
-                        EventTypeIndexes = default!, // configured later on
-                        EventSborTypeKinds = default!, // configured later on
+                        Events = new ReceiptEvents(), // configured later on
```

Comment:
Hmmm... I'd still prefer to define it as `null` over here and overwrite it later on using:
```
ledgerTx.Events = new ReceiptEvents(...)
```

EDIT: maybe even `ledgerTx.Events = ReceiptEvents.CreateFrom(events);` and encapsulate 213 LINQ `.Select(..).ToArray()`s inside this factory method?